.github/workflows/sub-ci-integration-tests-gcp.yml

```
@@ -82,7 +79,7 @@ jobs:
       app_name: zebrad
       test_id: sync-to-mandatory-checkpoint
       test_description: Test sync up to mandatory checkpoint
-      test_variables: "NETWORK=${{ inputs.network || vars.ZCASH_NETWORK }},NEXTEST_PROFILE=sync-to-mandatory-checkpoint,ZEBRA_TEST_SYNC_TO_CHECKPOINT=1,ZEBRA_CACHE_DIR=/home/zebra/.cache/zebra"
+      test_variables: "ZEBRA_NETWORK__NETWORK=${{ inputs.network || vars.ZCASH_NETWORK }},NEXTEST_PROFILE=sync-to-mandatory-checkpoint,TEST_SYNC_TO_CHECKPOINT=1"
```

This is running the `sync_to_mandatory_checkpoint_{mainnet, testnet}()` tests, but both will be testing whichever network is defined here.

Is the `ZEBRA_NETWORK__NETWORK` env var defined here expected or being used by this job anywhere?

We could split it up into two jobs if it is expected somewhere.

```suggestion
      test_variables: "NEXTEST_PROFILE=sync-to-mandatory-checkpoint,TEST_SYNC_TO_CHECKPOINT=1"
```

---

I preferred to segregate the tests in the nextest profile [`bcfce06` (#9768)](https://github.com/ZcashFoundation/zebra/pull/9768/commits/bcfce06222f794233abe7aaf9ac51967b09f5b22) to be able to make manual runs of this workflow for Testnet. It was also a bad idea to run this together.